sources/web/datalab/polymer/modules/file-manager/file-manager.ts

```diff
@@ -42,9 +42,14 @@ interface Notebook {
 }
 
 enum DatalabFileType {
-  DIRECTORY,
-  FILE,
-  NOTEBOOK,
+  DIRECTORY = 'directory',
+  FILE = 'file',
+  NOTEBOOK = 'notebook',
+}
+
+enum DatalabFileStatus {
+  IDLE = '',
+  RUNNING = 'Running',
```

Comment:
Are you directly using this string to display in the UI? Normally I would say that's a bad idea, but we are so far from having an internationalized UI that I don't know that it matters.